src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -173,7 +173,7 @@ public enum ConfigKey
             this.keyName=keyName;
         }
 
-        String getKeyName()
+        public String getKeyName()
```

Comment:
This is mainly used to avoid repeating string key names for the following two classes. We don't really have EncryptionOption object to call `toString()` on in this case.
![image](https://github.com/user-attachments/assets/22071908-a374-4dee-be15-dda6466c9b7a)

Reply:
While you are deciding on the `getKeyName()` change based on latest comment, I can look at the constructor change for spaces.

Reply:
I fixed the constructor spaces around `=` and the `for` loop braces.

Reply:
@maulin-vasavada so what if we replaced all calls of `getKeyName()` with `toString()` ?

Reply:
Will do.